rules/windows/process_creation/proc_creation_win_create_link_osk_cmd.yml

```diff
@@ -16,13 +16,13 @@ logsource:
 detection:
     selection_img:
         - Image|endswith: '\cmd.exe'
-        - OriginalFileName: 'Cmd.exe'
+        - OriginalFileName: 'Cmd.Exe'
```

Comment:
I kown it should not change the detection , but for me `modified` is need as the detection section is edited.